Readme.md

```diff
@@ -38,6 +38,11 @@ app.use(logger())
   Recommended that you `.use()` this middleware near the top
   to "wrap" all subsequent middleware.
 
+## Options
+
+* `logger` - an object implementing, at least, a `log` function with variadic arguments. (default `console`)
```

Comment:
If we went this route I'd almost prefer just passing objects so it's more of an event (or just use an emitter), but then that's effectively a new middleware if you were to re-implement the handler for those events, since the rest of the logic in here becomes more or less irrelevant to the new handler